plugins/kubernetes-backend/api-report.md

```diff
@@ -322,6 +324,14 @@ export type KubernetesObjectTypes =
   | 'statefulsets'
   | 'daemonsets';
 
+// @alpha
+export interface KubernetesRejectionHandler {
+  onRejected: (
+    reason: any,
```

Comment:
Should this really be using `any`? We try to avoid it as much as possible.

Reply:
This function gets used as the `onrejected` parameter in `Promise.catch` in a few places, which takes a parameter of type `any` [in the spec](https://github.com/microsoft/TypeScript/blob/896b9228710941f875294ebee2da998d500c8614/src/lib/es5.d.ts#L1515-L1520) -- it's too bad promises don't have an error type like `Exception` in Haskell.

Through super careful reasoning about `@kubernetes/client-node` it could become _sort of_ possible to put a type on it, but there's always a possibility that a crazy exception gets thrown and I'd lean towards making that clear in the type signature (and maybe an explanatory comment).

If you're _really_ against it I'll stare at all the code paths and try to figure out a sensible type, it just might take a while.

Reply:
Alright. Let's not dwell too much on it then. Does `unknown` work then?

Reply:
There's no more throw-and-catch stuff in the new PR!